.github/workflows/ci.yml

```diff
@@ -7,10 +7,11 @@ jobs:
     runs-on: ubuntu-latest
 
     steps:
-    - uses: actions/checkout@v1
+    - uses: actions/checkout@v3
     - name: Set up JDK
-      uses: actions/setup-java@v1
+      uses: actions/setup-java@v3
       with:
         java-version: 13
+        distribution: temurin
```

Comment:
@sullis ,

the `temurin` distribution does not seem to support java13, which is required for tests, 

see https://github.com/reactor/BlockHound/actions/runs/5391516225/jobs/9788493900?pr=361

could you revise your PR with another distribution ? or maybe remove the temurin distrib for jdk13 ?

thank you.